### ISSUE TITLE
Using metrics apart from accuracy

### ISSUE THREAD

**Issue by veddandekar** [xuxingya/tf2crf#31]
Keras allows using metrics apart from accuracy when compiling the model as below -

`model.compile(
    optimizer='adam',
    loss='mean_squared_error',
    metrics=[
        metrics.MeanSquaredError(name='my_mse'),
        metrics.AUC(name='my_auc'),
    ]
)`

However, I came to notice that when using tf2crf, no matter what metrics I specify while compiling the model, it only displays the accuracy metric during training.

Is this the expected behaviour? If so, any chance of implementing support for different metrics in a future release?

**Comment by xuxingya** [comment]
That is because it uses a customed training and testing step. To use different metrics, you can override the train_step and test_step function in the class.
